content/en/docs/tutorials/security/seccomp.md

```diff
@@ -32,7 +32,7 @@ profiles that give only the necessary privileges to your container processes.
 * Observe behavior when a missing profile is specified
 * Observe a violation of a seccomp profile
 * Learn how to create fine-grained seccomp profiles
-* Learn how to apply a container runtime default seccomp profile
+* Learn how to apply the default seccomp profile to a container
```

Comment:
The original text is correct, I think.
It refers to this section (https://kubernetes.io/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads). Those profiles are runtime default seccomp profiles.